.github/workflows/label-templated-discussions.yml

```diff
@@ -0,0 +1,97 @@
+name: Label Templated Discussions
+
+on:
+  discussion:
+    types: [created]
+
+jobs:
+  label-templated-discussion:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Get selected topic
+        id: get_selected_topic
+        uses: actions/github-script@v6
+        env:
+          DISCUSSION_HTML: ${{ github.event.discussion.html }}
+        with:
+          script: |
+            try {
+              const headings = $DISCUSSION_HTML.getElementsByTagName("h3");
```

Comment:
Would it be helpful for the the category form template to set an expected `id` on the element we are looking for, so we can just look for that `id` and return if its not available?

Reply:
@gnfisher yes that would be ideal

Reply:
Unfortunately the ids in the template don't end up in the discussion body html, so we had to go back to looking for the topic based on it's position https://github.com/community/community/pull/38309/commits/c295521cd0d6f9285e1ed8fc529f314a255864ec